Src/FluentAssertions/Formatting/PropertyInfoFormatter.cs

```diff
@@ -18,13 +18,7 @@ public bool CanHandle(object value)
 
     public void Format(object value, FormattedObjectGraph formattedGraph, FormattingContext context, FormatChild formatChild)
     {
-        if (value is not PropertyInfo property)
-        {
-            formattedGraph.AddFragment("<null>");
-        }
-        else
-        {
-            formattedGraph.AddFragment($"{property.DeclaringType?.Name ?? string.Empty}.{property.Name}");
-        }
+        var property = (PropertyInfo)value;
+        formattedGraph.AddFragment($"{property.DeclaringType?.Name}.{property.Name}");
```

Comment:
```suggestion
        // ReSharper disable once PossibleNullReferenceException
        formattedGraph.AddFragment($"{property.DeclaringType.Name}.{property.Name}");
```
Or just disable the warning here?

Reply:
No, the type is nullable, therefore we must check for null.